contracts/impls/trustable/WitnetRequestBoardTrustableBase.sol

```diff
@@ -0,0 +1,694 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "../WitnetRequestBoardUpgradableBase.sol";
+import "../../data/WitnetBoardDataACLs.sol";
+import "../../interfaces/IWitnetRequestBoardAdmin.sol";
+import "../../interfaces/IWitnetRequestBoardAdminACLs.sol";
+import "../../libs/WitnetParserLib.sol";
+import "../../patterns/Payable.sol";
+
+/// @title Witnet Request Board "trustable" base implementation contract.
+/// @notice Contract to bridge requests to Witnet Decentralized Oracle Network.
+/// @dev This contract enables posting requests that Witnet bridges will insert into the Witnet network.
+/// The result of the requests will be posted back to this contract by the bridge nodes too.
+/// @author The Witnet Foundation
+abstract contract WitnetRequestBoardTrustableBase
```

Comment:
I think this contract should inherit `WitnetRequestBoard.sol` as to make sure before deploying  it that  it contains all the functions needed, or am I missing something?

Reply:
I mean before deploying WRBTDefault and WRBTBoba

Reply:
`WitnetRequestBoardTrustableBase` inherits from `WitnetRequestBoardUpgradableBase` from which WitnetRequestBoard is inherited. So in fact, `WitnetRequestBoardTrustableBase` actually inherits from `WitnetRequestBoard` :-)